aspnetcore/mvc/views/partial.md

```diff
@@ -36,19 +36,17 @@ Partial views are created like any other view: you create a *.cshtml* file withi
 
 ## Referencing a Partial View
 
-From within a view page, there are several ways in which you can render a partial view. The simplest is to use `Html.Partial`, which returns an `IHtmlString` and can be referenced by prefixing the call with `@`:
-
-[!code-cshtml[](partial/sample/src/PartialViewsSample/Views/Home/About.cshtml?range=9)]
-
-The `PartialAsync` method is available for partial views containing asynchronous code (although code in views is generally discouraged):
+From within a view page, there are several ways in which you can render a partial view. The recommended way is to use `Html.PartialAsync`, which returns an `IHtmlString` and can be referenced by prefixing the call with `@`:
 
 [!code-cshtml[](partial/sample/src/PartialViewsSample/Views/Home/About.cshtml?range=8)]
```

Comment:
Suggest updating all `.cshtml` files to use only the `*Async` methods. For example, this line and line 15 still use `Html.Partial(...)`.